app/modules/intelligence/agents_copy/chat_agents/langchain_agent.py

```
@@ -0,0 +1,182 @@
+import json
+import os
+import re
+from typing import Any, List, AsyncGenerator, Union
```

Ruff (0.8.2)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)